crawl-ref/source/dat/des/branches/orc.des

```diff
@@ -1399,6 +1399,433 @@ x1...1...1x
 xxx..@..xxx
 ENDMAP
 
+NAME:    nicolae_orc_all_along_the_watchtower
+TAGS:    transparent no_descent
+DEPTH:   Orc, !Orc:$
+KMONS:   1 = orc wizard / orc priest
+KMONS:   2 = orc warrior ; chain mail | plate armour . arbalest | longbow
+KFEAT:   - = iron_grate
+SHUFFLE: c1, !I, -Ic`
+SUBST:   } = }]), ! = 1
+NSUBST:  1 = 2=1 / 2=2 / 2=12``` / `
+KPROP:   l = no_cloud_gen
+KMASK:   `Ic-}])12 = opaque
+KMASK:   `Ic-}])12 = no_monster_gen
+MAP
+  ...@...
+ .........
+...lllll...
+..llI!Ill..
+..lIc1cIl..
+@.l!1}1!l.@
+..lIc1cIl..
+..llI!Ill..
+...lllll...
+ .........
+  ...@...
+ENDMAP
+
+NAME:    nicolae_orc_beogh_chapel
+TAGS:    transparent
```

Comment:
See https://github.com/crawl/crawl/pull/3326 for the refresher write-up on extra and decor.

This, nicolae_orc_crystal_seam, nicolae_orc_found_a_spring, nicolae_orc_hostile_takeover, nicolae_orc_quarried_blocks, and nicolae_orc_stalacmites should all be tagged extra. With the other current vaults that makes for 16, a decently close if not sufficient enough point to bother with decor semi-guarantees.